src/main/java/com/zaxxer/nuprocess/linux/LinuxProcess.java

```diff
@@ -64,13 +63,10 @@ public NuProcess start(List<String> command, String[] environment, Path cwd) {
 
       try {
          prepareProcess(command, environment, cwd);
-
          if (pid == -1) {
```

Comment:
I was tempted to remove this because it can never actually happen, as far as I can tell. If the JVM's `forkAndExec` fails to start the process, it always throws an exception--it doesn't return `-1` despite how the native code looks. But it doesn't hurt anything to have it here, so I left it.